src/transfer.ts

```diff
@@ -74,3 +74,33 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+export async function transferAsset(
+  { from, to, assetId, amount, transactionParams, clawbackFrom, note, ...sendParams }: TransferAssetParams,
+  algod: Algodv2,
+) {
+  const transaction = algosdk.makeAssetTransferTxnWithSuggestedParamsFromObject({
```

Comment:
unit tests are needed for this method too, refer to tests on transfer algos for inspiration. Should cover main edge cases:

- recepient is not opted in - should throw an error 
- clawback is specified, should perform a clawback from target
- assed id is invalid or sender does not hold or opted in to the asset - should throw an error